src/Slices/ServiceCatalog/UI/Page.test.tsx

```diff
@@ -191,3 +160,34 @@ test("GIVEN ServiceCatalog WHEN new environment selected THEN new query is trigg
     environment: env2,
   });
 });
+//TO ADD: move that test to Service details
```

Comment:
Is this meant to be a separate ticket? If yes, can you link it here? :D If no, can you resolve the comment :D

Reply:
I forgot to move it :)